conjur-api/User.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="User.cs" company="Conjur Inc.">
+//     Copyright (c) 2016-2018 Conjur Inc. All rights reserved.
```

Comment:
Since it's a new file, the copyright should probably be 2018 Cyberark Ltd.

Reply:
done